ksqldb-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java

```diff
@@ -364,14 +363,20 @@ Arguments build() {
           timestampColumnName = Optional.ofNullable(timestampColumnName).orElse(null);
         }
 
-        try {
-          Objects.requireNonNull(schemaFile, "Schema file not provided");
-          Objects.requireNonNull(keyFormat, "Message key format not provided");
-          Objects.requireNonNull(valueFormat, "Message value format not provided");
-          Objects.requireNonNull(topicName, "Kafka topic name not provided");
-          Objects.requireNonNull(keyName, "Name of key column not provided");
-        } catch (final NullPointerException exception) {
-          throw new ArgumentParseException(exception.getMessage());
+        if (schemaFile == null) {
+          throw new ArgumentParseException("Schema file not provided");
+        }
+        if (keyFormat == null) {
+          throw new ArgumentParseException("Message key format not provided");
+        }
+        if (valueFormat == null) {
+          throw new ArgumentParseException("Message value format not provided");
+        }
+        if (topicName == null) {
+          throw new ArgumentParseException("Kafka topic name not provided");
+        }
+        if (keyName == null) {
+          throw new ArgumentParseException("Name of key column not provided");
```

Comment:
I would prefer to deduplicate code by having a method like
```
void requireArgumentsNotNull(final Object argument, final String errorMessage) {
    if (argument == null) {
        throw new ArgumentParseException(errorMessage);
    }
}
```

But I am also fine with this.

Reply:
I'll land this and follow up separately